src/mpyl/cli/commands/health/checks.py

```diff
@@ -146,6 +171,25 @@ def __check_config(console, env_var, default, schema_path, name):
     if os.path.exists(path):
         console.check(f"Found {location}", success=True)
 
+        path, diff = check_upgrade_needed(path, upgraders)
+        pretty_diff = pretty_print(diff) if diff else ""
+        if pretty_diff == "":
+            console.check("Upgrade not necessary", success=True)
+        else:
+            console.check("Upgrade required", success=False)
+            console.print("Expected changes:")
+            console.print(pretty_diff)
+            if sys.stdout.isatty() and Confirm.ask("Upgrade now?"):
```

Comment:
How is this going to work in the pipeline? Wouldn't the pipeline get stuck if upgrade is needed?

Reply:
The `sys.stdout.isatty()` should prevent that from happening.